Automattic/jetpack
# Protect: Add a default "See all results" label to the mobile navigation button

This PR adds a default label to the Protect mobile navigation. 

This PR only includes the addition of the default label, this label will also be receiving some design updates in a future PR as a part of the next big iteration of the plugin (p1HpG7-gSG-p2).

#### Changes proposed in this Pull Request:

* Adds a `label` property to the default navigation button's `item` object.

#### Other information:


#### Jetpack product discussion



1199980337313591-as-1202594951621279/f

#### Does this pull request change what data or activity we track or use?



Nope!

#### Testing instructions:






* Open the Protect admin page on a site with no vulnerabilities, in a mobile-sized window.
* Verify that the default label for the navigation dropdown is shown as "See all results".
* Add threats with the Jetpack Debug plugin, or install a vulnerable plugin and regenerate your status report.
* Verify that the default label for the navigation dropdown is shown as "All vulnerabilities".
* Verify that the desktop navigation is unaffected.

#### Screenshots

Before

<img width="374" alt="Screen Shot 2022-07-22 at 5 22 59 PM" src="https://user-images.githubusercontent.com/10933065/180580047-2453d174-9ecd-4767-ae08-30374b436082.png">

After

<img width="377" alt="Screen Shot 2022-07-22 at 5 22 45 PM" src="https://user-images.githubusercontent.com/10933065/180580051-7d91ccb8-d73f-4aa3-a303-c1976c8307f1.png">
